### PR TITLE
Update omnibrowser.md

### PR DESCRIPTION
**Pull request by lelewang111**

### CODE REVIEW

**Comment by lelewang111** [comment]
Hi David, I have update the content, please check! thank you so much!

**Comment by NoopDog** [comment]
Thanks will review!

File: content/analyze/portals/analysis-portals/omnibrowser.md [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
+Datasets are constantly updated on OmniBrowser™ in order to support the constantly growing demands from a wide range of customers. As datasets and analytical functions continue to expand on the OmniBrowser™ platform, Abiosciences will develop and distribute more collection of tools that will make the analyses of big data more efficient and effective in order to advance the biomedical R&D.
+
+![OmniBrowser™](https://user-images.githubusercontent.com/106643461/173044684-c7b1b5c2-2933-49e3-b052-78e4569cc7d9.jpg)
 
-![OmniBrowser](../../_images/portals/omnibrowser.jpg)
 
 ## Contact
 
-Analytical BioSciences (<mailto:bd_china@abiosciences.com>)
+Abiosciences (<mailto:bd_china@abiosciences.com>)
```
Comment (frano-m) [inline content/analyze/portals/analysis-portals/omnibrowser.md:24]:
```suggestion
Abiosciences (<bd_china@abiosciences.com>)
```

**Comment by frano-m** [comment]
@NoopDog and @lelewang111 this looks good to me - just one small change - the `mailto:` is not required see https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses.

@NoopDog we need to make a ticket to fix the `site-map` `OmniBrowser` `path`.

**Review by frano-m — APPROVED** [review]

**Review by NoopDog — APPROVED** [review]

**Pull request merged**